openstack/waf_hw/v1/whiteblackip_rules/results.go

```diff
@@ -17,6 +17,17 @@ type WhiteBlackIP struct {
 	White int `json:"white"`
 	// Policy ID
 	PolicyID string `json:"policyid"`
+
+	Name        string   `json:"name"`
+	Status      int      `json:"int"`
+	Description string   `json:"description"`
+	IPGroup     *IPGroup `json:"ip_group"`
+}
+
+type IPGroup struct {
+	GroupID   string `json:"id"`
+	GroupName string `json:"name"`
+	GroupSize int    `json:"size"`
```

Comment:
```suggestion
	ID   string `json:"id"`
	Name string `json:"name"`
	Size int    `json:"size"`
```
without "Group" in the **IPGroup** struct